test/Feature/WaveOps/WaveActiveCountBits.test

```diff
@@ -6,16 +6,12 @@ RWStructuredBuffer<uint> Out : register(u1);
 void main(uint3 threadID : SV_DispatchThreadID) {
   bool B1 = false;
 
-  switch (value[threadID.x]) {
-    case 0: // threads 0 and 1; result is number of active lanes (2)
-      Out[threadID.x + 4] = WaveActiveCountBits(true); // threads 0 and 1
-    case 2:
-      B1 = true; // set b1 to true for thread 3
-      break;
-    default:
-      Out[threadID.x + 4] = WaveActiveCountBits(false); // thread 2; expect 0
-      break;
-  }
+  if (value[threadID.x] == 0)
+    Out[threadID.x + 4] = WaveActiveCountBits(true); // threads 0 and 1
+  if (value[threadID.x] == 0 || value[threadID.x] == 1)
+    B1 = true; // set b1 to true for thread 3
```

Comment:
This sets `B1` to true for threads 0, 1, and 2, but not 3, due to LUT `value` being `[0, 0, 1, 2]`.